### ISSUE TITLE
Jupyterlab command line entrypoint should just be 'jupyterlab'

### ISSUE THREAD

**Issue by asmeurer** [jupyterlab/jupyterlab#3906]
When I install `jupyterlab`, I expect the command line entrypoint to be the same as the package, just `jupyterlab`, with no spaces or dashes. The `jupyter-lab`/`jupyter lab` entry points are also fine, but I think `jupyterlab` should also be an entrypoint, since that is what the project bills itself as.

**Comment by saulshanabrook** [comment]
I believe the entrypoint is `jupyter lab`/`jupyter-lab` because it is a [`jupyter` subcommand](http://jupyter.readthedocs.io/en/latest/projects/jupyter-command.html#description). So it uses the same [configuration approach](http://jupyter.readthedocs.io/en/latest/projects/config.html) and extends the built in server.

**Comment by jasongrout** [comment]
note that there is also precedent here - the `notebook` package has `jupyter notebook` as the starting command, for example.

**Comment by jasongrout** [comment]
Thanks - valuable feedback. Perhaps even just an alias would be helpful, though I have some reservation with breaking the existing pattern.

**Comment by asmeurer** [comment]
The existing entrypoints should definitely be kept intact.

**Comment by brichet** [comment]
> Two nice things about keeping things the way they were, and not making a new entrypoint:
> 
> 1. There is only one official way to launch jupyterlab: `jupyter lab`, which is simpler messaging and consistent with the other jupyter ecosystem tools
> 2. With `jupyter lab`, we have a layer of indirection through the jupyter_core command, which provides a central place for us to do things if we need to add some consistent logic across all of the jupyter tooling. If people get in the habit of doing `jupyterlab` instead of `jupyter lab`, we lose that indirection layer we have with all of our other tools.
> 
> I'd be +1 on *not* having a `jupyterlab` entry point for these reasons.

_Originally posted by @jasongrout in https://github.com/jupyterlab/jupyterlab/issues/18158#issuecomment-3605929762_

**Comment by jasongrout** [comment]
The third reason I'd prefer to keep just `jupyter lab` is that it's now been almost 8 years since the original request. By now `jupyter lab` is pretty entrenched, so perhaps it is not such a concern as when lab was first released? Or has this been a persistent annoyance for the last 8 years? Given that there has been no other comment on this issue until a few weeks ago, my guess is that it is no longer a concern.

In other words, did this hypothesis hold out in practice and is it still a concern?

> So I anticipate a very large number of users will do exactly what I did and try to start it with that command.

**Comment by brichet** [comment]
Thank for your thoughts @jasongrout

**Comment by brichet** [comment]
Closing this issue, feel free to reopen it if this is still a concern.
